tests/rptest/redpanda_cloud_tests/omb_validation_test.py

```diff
@@ -267,46 +277,84 @@ def test_max_connections(self):
             ],
         }
 
+        rejected_start = self._rejected_count()
+
+        def assert_no_rejected():
+            """Assert that there have been no rejected connections since the
+            start of the test."""
+            rejected_now = self._rejected_count()
+            assert rejected_now == rejected_start, f"Rejected connections detected, start={rejected_start}, now={rejected_now}"
+
         # ProducerSwarm parameters
         #
 
-        producer_kwargs = {}
-        producer_kwargs['min_record_size'] = 64
-        producer_kwargs['max_record_size'] = 64
+        record_size = 64
 
-        effective_msg_size = producer_kwargs['min_record_size'] + (
-            producer_kwargs['max_record_size'] -
-            producer_kwargs['min_record_size']) // 2
+        # very rough estimated number of connections for OMB
+        omb_connections = 3 * (omb_producer_count + omb_consumer_count)
+        # The remainder of our connection budget after OMB connections are accounted for we will
```

Comment:
Maybe expand the comment a little bit about what our connection budget actually is, i.e.: what the advertised tier connection limit means.

Reply:
Added a comment in https://github.com/redpanda-data/redpanda/compare/5d9ad58e47691dbc16c82a9e1b5b413cab4307e0..837694d18838e6329dab858c39c2e930e831ccfc